tests/test_components.py

```diff
@@ -771,5 +808,115 @@ def test_ice_sheet_too_high():
     assert 'exceeds maximum value' in str(excinfo.value)
 
 
+class TestGfsDycore(ComponentBase):
+    def get_component_instance(self, state_modification_func=lambda x: x):
+        return GfsDynamicalCore(number_of_longitudes=68,
+                                number_of_latitudes=32)
+
+    def get_3d_input_state(self):
+
+        component = self.get_component_instance()
+        state = climt.get_default_state(
+            [component], x=component.grid_definition['x'],
+            y=component.grid_definition['y'],
+            mid_levels=component.grid_definition['mid_levels'],
+            interface_levels=component.grid_definition['interface_levels'])
+
+        dcmip = climt.DcmipInitialConditions()
+        out = dcmip(state, add_perturbation=True)
+        state.update(out)
+
+        return state
+
+    def test_1d_output_matches_cached_output(self):
+        assert True
+
+
+class TestGfsDycoreWithPrognostic(ComponentBase):
+    def get_component_instance(self, state_modification_func=lambda x: x):
+        dycore = GfsDynamicalCore(number_of_longitudes=68,
+                                  number_of_latitudes=32)
+        radiation = RRTMGLongwave()
+        dycore.prognostics = [radiation]
+
+        return dycore
+
+    def get_3d_input_state(self):
+
+        prognostic = RRTMGLongwave()
+        component = self.get_component_instance()
+        state = climt.get_default_state(
+            [component, prognostic], x=component.grid_definition['x'],
+            y=component.grid_definition['y'],
+            mid_levels=component.grid_definition['mid_levels'],
+            interface_levels=component.grid_definition['interface_levels'])
+
+        dcmip = climt.DcmipInitialConditions()
+        out = dcmip(state, add_perturbation=True)
+        state.update(out)
+
+        return state
+
+    def test_1d_output_matches_cached_output(self):
```

Comment:
Write a comment here explaining that you're doing this to disable this test.

Reply:
done.